subsys/bluetooth/mesh/rpl_storage.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/** @brief Store an RPL entry.
+ *
+ * @param[in] rpl RPL entry to store.
+ * @param[in] force Flag for forcing the storage.
+ */
+void bt_mesh_rpl_strg_store(struct bt_mesh_rpl *rpl, bool force);
```

Comment:
Could you explain the usage of the `force` parameter a bit more, since you don't seem to be using it in any way in the implementation?

Another thing I'm not quite happy about is that you use `storage` in the filename and `STORAGE` in Kconfig, but `strg` in the API. I'm not a big fan of long symbol names, but would it be that bad to keep this consistent and spell out the whole word here as well?

Reply:
And now I see there was a bug in `bt_mesh_rpl_storage_store` aswell